### PR TITLE
Bioschemas changes by Noura

### PR DESCRIPTION
**Pull request by NishaSharma14**

### CODE REVIEW

File: app/Http/Controllers/ApplicationController.php
Comment (CS76) [inline app/Http/Controllers/ApplicationController.php:1]: unrelated commit

File: app/Jobs/ArchiveStudy.php
Comment (CS76) [inline app/Jobs/ArchiveStudy.php:1]: unrelated commit

File: composer.json
Comment (CS76) [inline composer.json:1]: unrelated commit

File: resources/js/App/Onboarding.vue
Comment (CS76) [inline resources/js/App/Onboarding.vue:1]: unrelated commit

File: resources/js/Pages/Publish.vue
Comment (CS76) [inline resources/js/Pages/Publish.vue:1]: unrelated commit

File: resources/js/Pages/Upload.vue
Comment (CS76) [inline resources/js/Pages/Upload.vue:1]: unrelated commit

File: resources/js/Shared/FileSystemBrowser.vue
Comment (CS76) [inline resources/js/Shared/FileSystemBrowser.vue:1]: unrelatead

File: resources/js/Shared/Primer.vue
Comment (CS76) [inline resources/js/Shared/Primer.vue:1]: unrelated

**Review by CS76 — CHANGES_REQUESTED** [review]

File: app/Http/Controllers/API/Schemas/Bioschemas/BioschemasController.php [hunk 1]
```diff
@@ -1,9 +1,9 @@
 <?php
 
-namespace App\Http\Controllers\API\Schemas\Bioschema;
+namespace App\Http\Controllers\API\Schemas\Bioschemas;
 
 use App\Http\Controllers\Controller;
-use App\Models\Bioschema\BioSchema;
+use App\Models\Bioschemas\BioSchemas;
```
[the remark this answers is not in the document]
Comment (NRayya) [reply app/Http/Controllers/API/Schemas/Bioschemas/BioschemasController.php:6]: It is a naming issue. The correct name is BioSchemas with "s"
